studygroups/api_urls.py

```diff
@@ -27,6 +28,5 @@
     url(r'^teams/members/$', views.TeamMembershipListView.as_view(), name='api_team_memberships'),
     url(r'^teams/invitations/$', views.TeamInvitationListView.as_view(), name='api_team_invitations'),
     url(r'^facilitator/invitations/$', views.facilitator_invitation_notifications, name='api_facilitator_invitations'),
-    url(r'^announcements/$', views.AnnouncementListView.as_view(), name='api_announcements'),
```

Comment:
@s-kennedy not a bit issue, but I recommend leaving the trailing `'` in places where the syntax allows - it makes seeing actual differences just a little bit easier, ie, if you add a URL at the end you only change 1 line, not 2